platform-sdk/swirlds-common/src/main/java/com/swirlds/common/metrics/platform/DefaultPlatformMetrics.java

```diff
@@ -44,7 +44,7 @@
 /**
  * Default implementation of the {@link Metrics} interface.
```

Comment:
I assume the name is temporary. Instead of doing a big bang migration I extract everything that can be easily extracted. That ends in 2 different metric implementations for some metrics and I need to give them names. That's why I ended with PlatformMetrics <- They are specific for platform-core